addons/mail/static/src/model/model_field_command.js

```diff
@@ -9,32 +9,44 @@ odoo.define('mail/static/src/model/model_field_command.js', function (require) {
 class FieldCommand {
     /**
      * @constructor
-     * @param {function} func function to call when executing this command. 
-     * The function should ALWAYS return a boolean value 
-     * to indicate whether the value changed.
+     * @param {string} name command name.
+     * @param {Object|Object[]} [data]
+     * data object or data objects array used for the command.
      */
-    constructor(func) {
-        this.func = func;
+    constructor(name, data) {
+        this._name = name;
+        this._data = data;
     }
 
     /**
-     * @param {ModelField} field
-     * @param {mail.model} record
-     * @param {options} [options]
-     * @returns {boolean} whether the value changed for the current field
+     * @returns {string}
      */
-    execute(field, record, options) {
-        return this.func(field, record, options);
+    get name() {
+        return this._name;
+    }
+
+    /**
+     * @returns {Object|Object[]}
+     */
+    get data() {
+        return this._data;
     }
 }
 
 /**
  * Returns a clear command to give to the model manager at create/update.
  */
 function clear() {
-    return new FieldCommand((field, record, options) =>
-        field.clear(record, options)
-    );
+    return new FieldCommand('clear');
+}
+
+/**
+ * Return a create command to give to the model manager at create/update.
+ *
+ * @param {Object|Object[]} data - data object or data objects array to create record(s)
+ */
```

Comment:
Note: all of these docstring are missing their `@returns` value

Also we should update them all to have a better comment, which is also explaining what is their goal.